code/modules/research/experimentor.dm

```diff
@@ -291,8 +291,9 @@
 			for(var/turf/T in oview(1, src))
 				if(!T.density)
 					if(prob(EFFECT_PROB_VERYHIGH))
-						var/obj/effect/decal/cleanable/reagentdecal = new/obj/effect/decal/cleanable/greenglow(T)
-						reagentdecal.reagents.add_reagent("radium", 7)
+						var/datum/reagents/R = new/datum/reagents(7)
```

Comment:
Just have it make a green glow and don't bother doing anything else.

The glow having reagents in it doesn't make a ton of sense, to be honest--nor is it particularly novel.

Making reagents to add a reagent to then cause a reagent to create a decal is just adding unnecessary steps when the reagents involved here don't really do anything.